src/pages/installation/cdn.md

```diff
@@ -10,44 +10,121 @@ contributors:
 
 # Ionic Packages
 
-Depending on whether you're using Angular, another framework, or none at all, there are a few different ways to install Ionic.
+Ionic provides different packages that can be used to quickly get started using Ionic Framework or Ionicons in a test environment, Angular, any other framework, or none at all.
 
-## Using Ionic in Angular
+## Ionic Framework CDN
 
-When using Ionic in an Angular project, install the latest `@ionic/angular` package from [npm](/docs/faq/glossary#npm). This comes with all of the Ionic components and Angular specific services and features.
+Ionic Framework can be quickly included from a CDN for quick testing in a [Plunker](https://plnkr.co/), [Codepen](codepen.io), or any other online code editor!
+
+It's recommended to use [unpkg](https://unpkg.com) to access the Framework from a CDN. To get the latest version, add the following inside the `<head>` element in an HTML file, or where external assets are included in the online code editor:
+
+```html
+<script src="https://unpkg.com/@ionic/core@latest/dist/ionic.js"></script>
+<link href="https://unpkg.com/@ionic/core@latest/css/ionic.bundle.css" rel="stylesheet">
+```
+
+With this it's possible to use all of the Ionic Framework core components without having to install a framework. The CSS bundle will include all of the Ionic [Global Stylesheets](../layout/global-stylesheets).
+
+<blockquote>
+  <p>
+    This does not install Angular or any other frameworks. This allows use of the Ionic Framework core components without a framework.
+  </p>
+</blockquote>
+
+
+## Angular + Ionic Framework
+
+When using Ionic Framework in an Angular project, install the latest `@ionic/angular` package from [npm](/docs/faq/glossary#npm). This comes with all of the Ionic Framework components and Angular specific services and features.
 
 ```shell
 $ npm install @ionic/angular@latest --save
```

Comment:
Do we need to say `@latest` here? That is [the default](https://docs.npmjs.com/cli/install).

Reply:
Without including `latest` it will install the latest or the installed version if it finds one. I'd rather include it to be verbose on the off chance it's already in the project.

```
$ npm outdated @ionic/angular
Package         Current  Wanted  Latest  Location
@ionic/angular    4.0.0   4.0.0   4.1.2  ionic-conference-app
```

```
$ npm i @ionic/angular
+ @ionic/angular@4.0.0
```

```
$ npm outdated @ionic/angular
Package         Current  Wanted  Latest  Location
@ionic/angular    4.0.0   4.1.2   4.1.2  ionic-conference-app
```

```
$ npm ls @ionic/angular
└── @ionic/angular@4.0.0
```